src/lib.rs

```diff
@@ -58,6 +58,12 @@ pub macro slab {
 mod test {
     use super::slab;
 
+    #[test]
+    fn slab_macro_can_create_empty_slab() {
+        let (slab, []): (slab::Slab<i32>, [usize; 0]) = slab![];
```

Comment:
I believe the second type in the tuple could be omitted like so: (slab::Slab<i32>, _)

Reply:
That's how I first wrote it, however:
![image](https://user-images.githubusercontent.com/9047632/47777660-8027f280-dcf5-11e8-964d-ea2cf21956eb.png)

Reply:
The type of `keys` is `[usize; element_count]`, where `element_count` is determined by `count_tts!()`, which is not a `const fn` on stable.

Reply:
I shall accept my defeat 😉

Reply:
If activate the feature `const_slice_len`, the tests don't compile, because the feature is not active for the tests, for some reason